firefly/kernel/memory-manager/primary/primary_phys.cpp

```diff
@@ -0,0 +1,25 @@
+#include "firefly/memory-manager/primary/primary_phys.hpp"
+
+#include "firefly/memory-manager/page.hpp"
+#include "firefly/memory-manager/primary/buddy.hpp"
+
+Pagelist pagelist;
+
+namespace firefly::kernel::mm::pmm {
+
+static BuddyManager buddy;
+
+void init(stivale2_struct_tag_memmap *mmap) {
+    buddy.init(std::move(mmap));
+    pagelist.init(std::move(mmap));
```

Comment:
whew you're starting to learn about move semantics huh? There's no point in moving a pointer tho :P

Reply:
heh yeah I am :sweat_smile: 
yes I see, I'll remove it right away :+1:

Reply:
All good, good that you’re learning